packages/tree-select/package.json

```diff
@@ -1,31 +1,31 @@
 {
-	"name": "@automattic/tree-select",
-	"version": "1.0.2",
-	"description": "A selector library for Redux",
-	"keywords": [
-		"redux",
-		"selector"
-	],
-	"author": "Ben Lowery <ben.lowery@automattic.com>",
-	"homepage": "https://github.com/Automattic/wp-calypso",
-	"license": "GPL-2.0-only",
-	"main": "dist/cjs/index.js",
-	"module": "dist/esm/index.js",
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/Automattic/wp-calypso.git"
-	},
-	"publishConfig": {
-		"access": "public"
-	},
-	"bugs": {
-		"url": "https://github.com/Automattic/wp-calypso/issues"
-	},
-	"files": [
-		"dist",
-		"src"
-	],
-	"dependencies": {
-		"lodash": "^4.17.0"
-	}
+    "name": "@automattic/tree-select",
+    "version": "1.0.2",
+    "description": "A selector library for Redux",
+    "keywords": [
+        "redux",
+        "selector"
+    ],
+    "author": "Ben Lowery <ben.lowery@automattic.com>",
+    "homepage": "https://github.com/Automattic/wp-calypso",
+    "license": "GPL-2.0-only",
```

Comment:
Just curious, why not to align license with Calypso that is "GPLv2 or later"?

https://github.com/Automattic/wp-calypso/blob/0dd4cace13cd7065cfb782da90ebf686747ce375/package.json#L6

Reply:
no good reason, we should change it.

Reply:
https://github.com/Automattic/wp-calypso/pull/29307